specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1106 @@
+{
```

Comment:
Are you using RPaaS for this new RP?

Reply:
We received approval for the RPaaS exception. In the image below, you'll see a similar question that the approver addressed.

![image](https://github.com/Azure/azure-rest-api-specs/assets/26116542/5256f1db-3943-44b7-b17d-e7cae6bcdc10)